verilog/formatting/tree_unwrapper.cc

```diff
@@ -990,7 +992,6 @@ static bool PartitionIsCloseParenSemi(const TokenPartitionTree& partition) {
 
 static bool PartitionIsCloseParen(const TokenPartitionTree& partition) {
   const auto ftokens = partition.Value().TokensRange();
-  if (ftokens.size() != 1) return false;
```

Comment:
Did this need to be removed?  It protects the following line against the case where `front()` does not exist.

Reply:
Actually in this specific case `ftokens.size()` returns 2. So is `if (ftokens.size() < 1) ` correct?

Reply:
Fixed